Files/Strings/en-US/Resources.resw

```diff
@@ -2691,4 +2691,24 @@
   <data name="BaseLayoutContextFlyoutNew.KeyboardAcceleratorTextOverride" xml:space="preserve">
     <value>Ctrl+Shift+N</value>
   </data>
+    <data name="SettingsPrivacy.Title" xml:space="preserve">
+    <value>Privacy policy</value>
+  </data>
+  <data name="SettingsFluentUISystemIconsLicenseButton.Content" xml:space="preserve">
+    <value>Fluent UI System Icons</value>
+  </data>
+  <data name="SettingsFluentUISystemIconsLicenseButton.AutomationProperties.Name" xml:space="preserve">
+    <value>Fluent UI System Icons</value>
+  </data>
+  <data name="SettingsPrivacyPolicyMarkdownBlock.Text" xml:space="preserve">
+    <value>
+**Privacy Policy**
```

Comment:
Should that really be here? What if privacy policy is updated and some languages didn't reflect the changes? Or maybe meaning and phrasing is altered in a translation? Wouldn't it be better if privacy policy was just pulled from github through an API interface?

Reply:
That's fair, but it seems unlikely the privacy policy will really change. And if it does, the existing translations can just be reset or marked as needs review so that translators know to update it. Having it in the app also allows it to work offline as well.

Also cc @yaichenbaum

Reply:
The privacy policy isn't really something that will be updated very often and like Winston said, we can update the strings so translators have to translate it again.